rossum/lib/api_client.py

```diff
@@ -355,6 +355,10 @@ def _get_annotation_polling(self, annotation: int) -> dict:
             click.echo(" Processing failed.")
         return annotation_object
 
+    def create_workspace(self, name: str, organization: str, queues: Optional[List] = None) -> dict:
```

Comment:
Based on the documentation, [queues](https://api.elis.rossum.ai/docs/#workspace) is only a read-only field. Why should user have an option to add some and how it would be performed in the request?

Reply:
Please make it possible to pass `metadata` in the request as well. We are now working on making `rossum` a robust API Client, so even though you may not need to pass this value in your current use cases, the functions should give a user an opportunity to pass all the data that they would be able to pass if using Rossum API directly, in their requests using `rossum`.

Reply:
thank you for the notice, I will remove queues, I also have checked the uptime-test, there is no need to pass the queue param for creation of a workspace